docs/collections/electric-collection.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Electric Collection
+---
+
+# Electric Collection
+
+Electric collections provide seamless integration between TanStack DB and ElectricSQL, enabling real-time data synchronization with your Postgres database through Electric's sync engine.
+
+## Overview
+
+The `@tanstack/electric-db-collection` package allows you to create collections that:
+- Automatically sync data from Postgres via Electric shapes
+- Support optimistic updates with transaction confirmation and automatic rollback on errors
+- Handle persistence through customizable mutation handlers
+
+## Installation
+
+```bash
+npm install @tanstack/electric-db-collection @tanstack/db
```

Comment:
You (almost) never want to install the db package. Instead you install a framework package which re-exports all the db stuff

Reply:
I saw that in the other references and was not sure if the idea was not to prescribe the framework. I'll change it to `react-db`